src/expression.d

```diff
@@ -3732,7 +3734,7 @@ public:
     Dsymbol s;
     bool hasOverloads;
 
-    extern (D) this(Loc loc, Dsymbol s, bool hasOverloads = false)
+    extern (D) this(Loc loc, Dsymbol s, bool hasOverloads = true)
```

Comment:
I don't think so. Half of `hasOverloads` would need to be set to `false` for `VarDeclaration`s. For them, explicit argument passing is just verbose. The default argument `true` works for good default behavior.